### ISSUE TITLE
Extra space in generics II

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#57]
With SwiftFormat 0.16.4,
```swift
let bufferSize = bufferLength * MemoryLayout<UInt8>.size
```
becomes
```swift
let bufferSize = bufferLength * MemoryLayout < UInt8>.size
```
It seems related to #52 and #54.

**Comment by nicklockwood** [comment]
Yay, more generics bugs! :-) Thanks for reporting, I'll fix it ASAP.

**Comment by nicklockwood** [comment]
@RabugenTom  This is working correctly for me. Are you sure you're running 0.16.4?

If so, can you attach the whole file (or a cut down version) that includes that line and is formatted incorrectly?

**Comment by RabugenTom** [comment]
Indeed, it's an inequality a few lines below that makes it fail. Here is a minimal failing example:
```swift
let a = Something<T>
let b = c > a
```
becomes
```swift
let a = Something < T>
let b = c > a
```
with 0.16.4 (confirmed).

**Comment by nicklockwood** [comment]
Thanks for the clarification! Fixed in 0.17.
